src/ga4gh/vrs/extras/vcf_annotation.py

```diff
@@ -1,122 +1,168 @@
-"""
-Annotate VCF files with VRS
-
-Input Format: VCF
-Output Format: VCF
-
-The user should pass arguments for the VCF input, VCF output, &
-the vrs object file name.
-
-ex. python3 src/ga4gh/vrs/extras/vcf_annotation.py input.vcf.gz --out
-./output.vcf.gz --vrs-file ./vrs_objects.pkl
-"""
-
-import argparse
-import sys
+"""Module containing tools for annotating VCFs with VRS"""
 import pickle
-import time
+from enum import Enum
+from typing import Dict, List, Optional
+from timeit import default_timer as timer
 
+import click
 from biocommons.seqrepo import SeqRepo
 import pysam
 
-from ga4gh.vrs.dataproxy import SeqRepoDataProxy
+from ga4gh.vrs.dataproxy import SeqRepoDataProxy, SeqRepoRESTDataProxy
 from ga4gh.vrs.extras.translator import Translator
 
 
-class VCFAnnotator:
-    """
-    This class provides utility for annotating VCF's with VRS allele id's.
+class SeqRepoProxyType(str, Enum):
+    """Define constraints for SeqRepo Data Proxy types"""
+
+    LOCAL = "local"
+    REST = "rest"
+
+
+@click.command()
+@click.option(
+    "--vcf_in",
+    required=True,
+    type=str,
+    help="The path for the input VCF file to annotate"
+)
+@click.option(
+    "--vcf_out",
+    required=True,
+    type=str,
+    help="The path for the output VCF file"
+)
+@click.option(
+    "--vrs_file",
+    required=True,
+    type=str,
+    help="The path for the output VCF pickle file"
+)
+@click.option(
+    "--seqrepo_dp_type",
+    required=False,
+    default=SeqRepoProxyType.LOCAL,
+    type=click.Choice([v.value for v in SeqRepoProxyType.__members__.values()],
+                      case_sensitive=True),
+    help="The type of the SeqRepo Data Proxy to use"
+)
+@click.option(
+    "--seqrepo_root_dir",
+    required=False,
+    default="/usr/local/share/seqrepo/latest",
+    help="The root directory for local SeqRepo instance"
+)
+@click.option(
+    "--seqrepo_base_url",
+    required=False,
+    default="http://localhost:5000/seqrepo",
+    help="The base url for SeqRepo REST API"
+)
+def annotate_click(vcf_in: str, vcf_out: str, vrs_file: str,
+                   seqrepo_dp_type: SeqRepoProxyType, seqrepo_root_dir: str,
+                   seqrepo_base_url: str) -> None:
+    """Annotate VCF file via click"""
```

Comment:
I think retaining the example command (`ex. python3 src/ga4gh/vrs/extras/vcf_annotation.py input.vcf.gz --out ./output.vcf.gz --vrs-file ./vrs_objects.pkl` from the previous commit) might be worthwhile -- you add a new line in here

Reply:
Probably would be good to include a README for the tool once it’s complete. Can do in #121

Reply:
Is that a findable location? I don't know if I'd expect to find an execution example in the `if __name__...` clause. If you put it in the docstring, then it's accessible via the `--help` message, but readme makes sense as well

Reply:
Ended updating in this branch since #121 will not be addressed in the project.